tests/test-bson.c

```diff
@@ -2317,5 +2336,6 @@ test_bson_install (TestSuite *suite)
                   "/bson/unsupported_type/empty_key",
                   test_bson_visit_unsupported_type_empty_key);
    TestSuite_Add (suite, "/bson/binary_subtype_2", test_bson_subtype_2);
+   TestSuite_Add (suite, "/bson/CDRIVER-2455", test_bson_CDRIVER_2455);
```

Comment:
Let's name it like the others. Perhaps "/bson/regex_length" and test_bson_regex_lengths.

Reply:
Done